runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,58 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({opts})                                         *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range. This method does not make an LSP request, but just queries locally
+    stored inlay hints. The returned inlay hints are unresolved.
+
+    Example usage: >lua
+        local hint = vim.lsp.inlay_hint.get({ bufnr = bufnr })[1]
+        resolved_hint = client.request(methods.inlayHint_resolve, hint.inlay_hint, callback, bufnr)
+        vim.lsp.util.apply_text_edits(resolved_hint.textEdits, bufnr, 'utf-8')
+        command = resolved_hint.label[1].command
+        vim.lsp.buf.execute_command({ command = command })
+<
+
+    Parameters: ~
+      • {opts}  vim.lsp.inlay_hint.get.opts ? Optional table which holds the following optional fields:
+                • bufnr (integer?): Only return inlay hints from the clients
+                  attached to the given buffer.
+                • filter (function?): Predicate used to filter clients.
+                  Receives a client as argument and must return a boolean.
+                  Clients matching the predicate are included. Example: >lua
+
+                     -- Don't request inlay hints from typescript-language-server
+                     vim.lsp.inlay_hint.get {
+                       filter = function(client) return client.name ~= "tsserver" end
+                     }
+<
+                • id (integer?): Only return inlay hints from the client with
+                  ID (client.id) matching this field.
+                • name (string?): Only return inlay hints from the client with
+                  name (client.name) matching this field.
+                • range (lsp.Range?): Only return inlay hints within this
```

Comment:
Feels kind of odd for this explanation of `lsp.Range` to be here instead of where it is defined.

Reply:
There is an existing longer hover doc for `lsp.Range`, but it doesn't show up in vimdoc :sob: so this is just tl;dr version

Reply:
Hmmm even if the explanation is relevant here, I feel we should exclude it. It just doesn't feel like the right place for this description to show up in.

Reply:
Makes sense, removed